db_setup/schema.sql

```diff
@@ -0,0 +1,109 @@
+USE plants;
+GO
+
+DROP TABLE if exists s_beta.watering, s_beta.duty, s_beta.recording, s_beta.botanist, s_beta.plant, s_beta.location;
+GO
+
+CREATE TABLE s_beta.recording (
+    "id" INT IDENTITY(1,1),
+    "plant_id" INT NOT NULL,
+    "soil_moisture" FLOAT NOT NULL,
+    "temperature" FLOAT NOT NULL,
```

Comment:
I genuinely don't know, because I've not been working with the data much, but does every plant have a reading for all of these? What if the hygrometer breaks, but the thermometer still works?

Reply:
(I mean in terms of the not nulls)

Reply:
I think we determined they can be left as NOT NULL in our discussion earlier as they appear to be consistently present.